src/mlo_dir_conv.cpp

```diff
@@ -142,7 +142,8 @@ static auto GetWindogradSolvers()
                                            miopen::solver::ConvMPBidirectWinograd_xdlops<3, 3>,
                                            miopen::solver::ConvMPBidirectWinograd_xdlops<4, 3>,
                                            miopen::solver::ConvMPBidirectWinograd_xdlops<5, 3>,
-                                           miopen::solver::ConvMPBidirectWinograd_xdlops<6, 3>>{};
+                                           miopen::solver::ConvMPBidirectWinograd_xdlops<6, 3>,
+                                           miopen::solver::ConvWinoFuryRxS<2, 3>>{};
```

Comment:
I don't know. Probably.
What are the limitations for WrW direction?

Reply:
I mean is there any other specific limitations rather than the normal ones?

Reply:
The limitations are the same, but as only filter size <= 3 is supported by Winograd Fury, for the WrW direction it will be cases where the output tensor sizes <= 3 (in the WrW direction, we swap filter and output tensors to get the equivalent forward parameters). 

Example: for WrW case `-W 24 -H 24 -c 16 -n 1 -k 16 -x 22 -y 22 -p 0 -q 0 -u 1 -v 1` equivalent forward convolution parameters would be `-W 24 -H 24 -c 1 -n 16 -k 16 -x 3 -y 3 -p 0 -q 0 -u 1 -v 1`.

It would be interesting to parse find-db and see if there are any cases like this.

Reply:
Let's disable this solver for the WrW direction for now.